scripts/docker-cross-tag.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024-2024  The DOSBox Staging Team
+
+usage() {
+    printf "%s\n" "\
+Usage: $0 FUNC OPTIONS
+
+Utility script to assist with docker image tags
+
+FUNC must be one of:
+  tag-exists         Test if image tag exists in GHCR.
```

Comment:
It would be better to spell it out:

```
Test if image tag exists in the GitHub Container Registry (GHCR).
``

That would give people unfamiliar with GHCR at least a trail to follow.